Resources/Core/Utilities/Money/README.md

```diff
@@ -0,0 +1,150 @@
+#  Money
+
+Handle monetary values with ease.  
+`Money` is a Swift implementation of the popular js library [Dinero.js](https://dinerojs.com/).
+
+## Overview  
+
+A Money is initialized with:
+- **cents**: `Integer` - value in minor currency units (eg. cents)
+- **currency**: `currency` - enum value of the supported currencies
+- **localeIdentifier**: Optional `String` - identifier for the Locale object that we use for the output formatting (eg. "en_US"). If not provided, we will use `Locale.current.identifier` as default value.
+- **precision**: Optional `Int` - number of decimal places to represent value, default value is 2.
+
+### Getters:
+- cents: Cents (typealias Cents = Int)
+- currency: Currency
+- localeIdentifier: String
+- precision: Int
+- unitValue: Double
+- formatted: String?
+- locale: Locale
+### Manipulation:
+- +, -, *, /
+- multiply()
+- divide()
+- percentage()
+- setLocaleIdentifier()
+### Muttating Manipulations:
+- multiply()
+- divide()
+- percentage()
+- setLocaleIdentifier()
+### Testing
+- ==, >, >=, <, <= 
+- isPositive
+- isNegative
+- isZero
+- hasSameAmount()
+- hasSameCurrency()
+---
+Additional: Confirmation to the protocols: Equatable, Comparable, Hashable, Codable, ExpressibleByIntegerLiteral, CustomStringConvertible    
+  
+---
+
+## Support types  
+
+We use couple additional types as a support for main `Money` struct:
+- typealias Cents = Int
+- enum `Currency`:
+``` swift
+public enum Currency: Codable {
+  /// U.S. Dollar (USD)
+  case usd
+  /// European Euro (EUR)
+  case eur
+  /// Canadian Dollar (CAD)
+  case cad
+  /// Chinese Yuan Renminbi (CNY)
+  case cny
+  /// Japanese Yen (JPY)
+  case jpy
+  /// British Pound (GBP)
+  case gbp
+  /// Swiss Franc (CHF)
+  case chf
```

Comment:
Should we add an `unknown` case for other currencies?

Reply:
Not sure 🤔
What will be the currency code and symbol for that case?

Reply:
Maybe just leave it as is. We'll add new currencies once we get to a situation.